fix(table): #1653

justwiner opened the pull request:
Fix table can not correctly calculate bug of cell width after sorting or filtering

fix #1653

Please make sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer to relative issues for your PR.

Ryan2128 commented on packages/table/src/table-layout.ts:
```diff
@@ -72,12 +72,19 @@ class TableLayout {
+      let scrollY = true
       const prevScrollY = this.scrollY.value
-      const scrollY = body.offsetHeight > this.bodyHeight.value
+      // bodyHeight无值时，表示未设置table高度，永远不会出现滚动条
```
Maybe English annotations are better?

justwiner replied:
Done

Commit:
fix(table): fix #1653

Fix table can not correctly calculate bug of cell width after sorting or filtering

fix #1653
